.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Is this a bug report or feature request?**
+* Bug Report
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How to Reproduce**
+Steps to reproduce the behavior:
+1. Deploy AuthService ...
+2. Perform this action ...
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Config Files**
+Please provide all relevant configuration. This includes:
+- AuthService configuration.
+- OIDC Provider configuration.
+
+If relevant, upload your configuration files here using GitHub, there is no need
+to upload them to any 3rd party services
+
+**Logs**
+Please provide all relevant logs (e.g., AuthServicelog , OIDC Provider logs, etc.)
+
+**Environment:**
+- AuthService version: (found in image tag)
+- Platform: (GKE, Azure, minikube, custom...) 
+- Kubernetes version: 
```

Comment:
There's trailing whitespace in these two lines. May I suggest the following:

1. A rule to highlight trailing whitespace in your editor.
2. Add:

   ```
   [color]                                                                                                                                                                                                                                        
   ui = true  
   ```

to your `.gitconfig`, which highlights the trailing whitespace by default.

Reply:
Added an extension to my VSCode.
In the future, I'd like to find (or write) a linter for this and add it as a GitHub action in our workflow.